chocolatey-community/chocolatey-packages
# (freevideoeditor) Partially embed package

## Description

The package is moved to be partially embedded with this PR, as well as upgrading the icon of the package.

## Motivation and Context

The package has become too large, and can no longer be pushed to Chocolatey Community Repository  while it is fully embedded. As such it needs to change away from a fully embedded package.

## How Has this Been Tested?

1. Run `.\update_all.ps1 -ForcedPackage freevideoeditor freevvideoeditor`
2. Install the package normally and verify it installed the 64bit executable.
3. Uninstall the Package.
4. Install the package while passing in `--x86` and verify it downloads and installs using the 32bit executable.

## Screenshot (if appropriate, usually isn't needed):

## Types of changes


## Checklist:
